bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -125,19 +125,19 @@ $ConfigRole
         YES --> $SecondaryStateDecider
             CORNER_KICK --> $SecondaryStateTeamDecider
                 OUR --> $RankToBallNoGoalie + use_time_to_ball:true
-                    FIRST --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @AvoidBallActive, @GoToCornerKickPosition + mode:striker
-                    SECOND --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @AvoidBallActive, @GoToCornerKickPosition + mode:supporter
-                    THIRD --> @ChangeAction + action:positioning, @AvoidBallActive, @GoToDefensePosition
-                OTHER --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @AvoidBallActive, @GoToCornerKickPosition + mode:others
+                    FIRST --> @ChangeAction + action:positioning, @ForgetBall, @LookAtFieldFeatures, @AvoidBallActive, @GoToCornerKickPosition + mode:striker
+                    SECOND --> @ChangeAction + action:positioning, @ForgetBall, @LookAtFieldFeatures, @AvoidBallActive, @GoToCornerKickPosition + mode:supporter
+                    THIRD --> @ChangeAction + action:positioning, @ForgetBall, @LookAtFieldFeatures, @AvoidBallActive, @GoToDefensePosition
```

Comment:
Is there any reason, `@LookAtFieldFeatures` was not set before?

Reply:
that was wrong, thanks for finding it